src/schematics/workspace/files/BUILD.bazel__tmpl__

```diff
@@ -0,0 +1,40 @@
+package(default_visibility = ["//visibility:public"])
+exports_files(["tsconfig.json"])
+
+load("@build_bazel_rules_angular//:defs.bzl", "ng_external_libraries")
+
+filegroup(name = "node_modules", srcs = glob([
+    # should not be whitelisted
+    "node_modules/@angular/**",
+    "node_modules/typescript/**",
+    "node_modules/zone.js/**",
+    "node_modules/rxjs/**/*.d.ts",
+    "node_modules/rxjs/**/*.js",
+    "node_modules/@types/**/*.d.ts",
+    "node_modules/tsickle/**",
+    "node_modules/hammerjs/**/*.d.ts",
+    "node_modules/protobufjs/**",
+    "node_modules/bytebuffer/**",
+    "node_modules/reflect-metadata/**",
+    "node_modules/minimist/**/*.js",
+    "node_modules/webpack/**"
+], exclude=["node_modules/@angular/cli/**"]))
+
+ng_external_libraries(name = "ng_libs", srcs = glob([
+    "node_modules/@angular/**"
+], exclude = [
+    "node_modules/@angular/cli/**",
+    "node_modules/@angular/platform-browser/animations*",
+    "node_modules/@angular/platform-browser/animations/**",
+
+    # Alex E?
```

Comment:
this should go away soon when we do bootstrap codegen in the `scripts.postinstall`